docs/known_issues.rst

```diff
@@ -75,6 +75,39 @@ Both will throw an exception if units do not cancel, e.g.::
 
 See: https://github.com/astropy/astropy/issues/7582
 
+Quantities Lose Their Units with Operations on pandas.Series
+------------------------------------------------------------
+
+Quantities may work with certain operations on `~pandas.Series` but
+this behaviour is not tested. Quantities are subclassed from ``numpy``'s `~numpy.ndarray`
+and they do not behave same as `~pandas.Series` objects.
+For example one know issue is that multiplying a `~pandas.Series` object
+with a unit will *not* return a ``Quantity`` object. It will return a `~pandas.Series`
+object without any unit::
```

Comment:
```suggestion
For example, multiplying a `~pandas.Series` instance
with a unit will *not* return a |Quantity|. It will return a `~pandas.Series`
object without any unit::
```